docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx

```diff
@@ -117,6 +119,16 @@ $ echo <Var name="version-server-url/path" />/<Var name="release-channel" /> | s
 Make sure not to include `https://` as a prefix to the server address.
 </Admonition>
 
+If the agent was not installed using the system package manager, follow the instructions on the
+[Teleport Enterprise Cloud Downloads page](../../choose-an-edition/teleport-cloud/downloads.mdx)
+to add the teleport package repository and reinstall teleport. To check if teleport
```

Comment:
TBH I find these instructions pretty hard to follow/understand right now because it's just one wall of text. I think we should refactor this guide a bit to follow our normal guide structure with steps e.g.

```
## Step 1. Install Teleport updater package
// explain that Teleport needs to be installed from apt/rpm repo and how to check

## Step 2. Create updater configuration
//

## Step 3. Check for updates
//
```

No new content is actually needed, just structuring what we have a bit better. What do you think?